idutils/config.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of IDUtils
+# Copyright (C) 2024 CERN.
+#
+# IDUtils is free software; you can redistribute it and/or modify
+# it under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Default configuration."""
+
+from .utils import *
+
+PID_SCHEMES = [
```

Comment:
Should we prepend all the config variables with IDUTILS_? (For consistency with the other modules/vars)

Reply:
Yes indeed, this was discussed, maybe forgotten?